# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/openvdb/hotfix-tbb-ver.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- openvdb-13.0.0/openvdb/openvdb/thread/Threading.h.vanilla	2026-09-01 21:00:43.465733648 +0300
+++ openvdb-13.0.0/openvdb/openvdb/thread/Threading.h	2026-09-01 21:01:19.097261796 +0300
@@ -34,7 +34,7 @@
 {
     // @note 12000 was the 2021.1-beta05 release. The 2021.1-beta08 release
     //   introduced current_context().
-#if TBB_INTERFACE_VERSION >= 12002
+#if 1
     auto ctx = tbb::task::current_context();
     return ctx ? ctx->cancel_group_execution() : false;
 #else
@@ -47,7 +47,7 @@
 {
     // @note 12000 was the 2021.1-beta05 release. The 2021.1-beta08 release
     //   introduced current_context().
-#if TBB_INTERFACE_VERSION >= 12002
+#if 1
     auto ctx = tbb::task::current_context();
     return ctx ? ctx->is_group_execution_cancelled() : false;
 #else

